src/renderer/components/TopTabs.tsx

```diff
@@ -37,7 +37,7 @@ const TopTabs = (): JSX.Element => {
           </Grid>
           <Grid item xs={2}>
             <Typography align="center" variant="subtitle2">
-              Version: {version}
+              Version: {cliVersion + ' ' + gitHash}
```

Comment:
We should probably still include the main version.  Something like:

Version: ${version}-${gitHash}
Pomerium-CLI Version: ${cliVersion}

Reply:
can you add it to the metadata then. it gives me warnings including from package.json

Reply:
i guess i can do it
also this is a design change and will look bad